dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -60,6 +61,43 @@ def filter
     render json: {error: e.message}, status: :bad_request
   end
 
+  # GET /api/v1/pd/workshops/upcoming_teachercon
+  # returns all upcoming teachercons for admins, associated teachercons for
+  # regional partners, and an empty set for everyone else.
+  def upcoming_teachercon
```

Comment:
nit - this should be `upcoming_teachercons`

Reply:
:+1: